apps/emqx_bridge/test/emqx_bridge_v2_testlib.erl

```diff
@@ -1241,13 +1241,29 @@ t_on_get_status(Config, Opts) ->
             ok;
         _ ->
             emqx_common_test_helpers:with_failure(down, ProxyName, ProxyHost, ProxyPort, fun() ->
-                ?retry(
-                    _Interval0 = 100,
-                    _Attempts0 = 20,
-                    ?assertEqual(
-                        {ok, FailureStatus}, emqx_resource_manager:health_check(ResourceId)
-                    )
-                )
+                case is_list(FailureStatus) of
+                    true ->
```

Comment:
Nit. To me, this case fits better inside the `?retry` block.

Reply:
It was already inside a retry before, and still fails a lot in CI.  :thinking: